Services/CO.CDP.Organisation.WebApi/Model/Command.cs

```diff
@@ -2,43 +2,37 @@
 
 namespace CO.CDP.Organisation.WebApi.Model;
 
-public record RegisterOrganisation
+public record NewOrganisation
```

Comment:
This was called "RegisterOrganisation" as it was treated as a command. It's still in the Command namespace, so has to either follow command naming conventions (an intended action to invoke), or the namespace should change.

Reply:
If you want to move away from the command pattern, that's fine, but that requires namespace change.